plugins/browser-plugin-ga-cookies/README.md

```diff
@@ -36,14 +36,25 @@ Initialize your tracker with the GaCookiesPlugin:
 import { newTracker } from '@snowplow/browser-tracker';
 import { GaCookiesPlugin } from '@snowplow/browser-plugin-ga-cookies';
 
-newTracker('sp1', '{{collector}}', { plugins: [ GaCookiesPlugin() ] }); // Also stores reference at module level
+newTracker('sp1', '{{collector}}', { plugins: [ GaCookiesPlugin(
+    /* pluginOptions */
+) ] });
+
+/* 
+ * Available plugin options `GACookiesPluginOptions`:
+ * {
+ * ua: Send Universal Analytics specific cookie values. Defaults to true.
+ * ga4: Send Google Analytics 4 specific cookie values. Defaults to true
+ * ga4ContainerId: Container id to search the Google Analytics 4 session cookie. The cookie has the form of _ga_<container-id> where <container-id> is the data stream container id. Defaults to the empty string.
+ * } 
+ */
```

Comment:
Can we update `trackers/javascript-tracker/src/configuration.ts` and `trackers/javascript-tracker/src/features.ts` to allow passing this through in the `newTracker` call too?